frontend/src/util/appInterface.js

```diff
@@ -1,38 +1,45 @@
-var previouslySelectedAppFeature = ''
-var previouslySelectedAppFullScreen = ''
-var APP_NOT_LOADED = 'appNotLoaded'
-var TIME_TO_RETRY = 500
-var RETRY_TIMEOUT = 60000
+import { CUSTOM_EVENT } from 'tracim_frontend_lib'
 
-var getSelectedApp = function (appName) {
+let previouslySelectedAppFeature = ''
+let previouslySelectedAppFullScreen = ''
+const APP_NOT_LOADED = 'appNotLoaded'
+const TIME_TO_RETRY = 500
+const RETRY_TIMEOUT = 60000
+
+const getSelectedApp = function (appName) {
   // FIXME - CH - 2019-06-18 - The try/catch is a temporary solution to solve the frontend, apps and appInterface.js
   // loading and execution order. If getSelectedApp return APP_NOT_LOADED, GLOBAL_renderAppFeature and GLOBAL_renderAppFullscreen
   // will retry every TIME_TO_RETRY ms for RETRY_TIMEOUT ms
   // see https://github.com/tracim/tracim/issues/1954
+
+  /*
+    global appHtmlDocument, appThread, appFile, appFolderAdvanced, appAdminWorkspaceUser, appWorkspaceAdvanced,
+    appAgenda, appShareFolderAdvanced, appCollaborativeDocumentEdition, appGallery, appWorkspace
+  */
   try {
     switch (appName) {
       case 'html-document':
-        return (appHtmlDocument || {default: {}}).default
+        return (appHtmlDocument || { default: {} }).default
       case 'thread':
-        return (appThread || {default: {}}).default
+        return (appThread || { default: {} }).default
       case 'file':
-        return (appFile || {default: {}}).default
+        return (appFile || { default: {} }).default
       case 'workspace':
-        return (appWorkspace || {default: {}}).default
+        return (appWorkspace || { default: {} }).default
       case 'folder':
-        return (appFolderAdvanced || {default: {}}).default
+        return (appFolderAdvanced || { default: {} }).default
       case 'admin_workspace_user':
-        return (appAdminWorkspaceUser || {default: {}}).default
+        return (appAdminWorkspaceUser || { default: {} }).default
       case 'workspace_advanced':
-        return (appWorkspaceAdvanced || {default: {}}).default
+        return (appWorkspaceAdvanced || { default: {} }).default
       case 'agenda':
-        return (appAgenda || {default: {}}).default
+        return (appAgenda || { default: {} }).default
       case 'share_folder':
-        return (appShareFolderAdvanced || {default: {}}).default
+        return (appShareFolderAdvanced || { default: {} }).default
       case 'collaborative_document_edition':
-        return (appCollaborativeDocumentEdition || {default: {}}).default
+        return (appCollaborativeDocumentEdition || { default: {} }).default
       case 'gallery':
-        return (appGallery || {default: {}}).default
+        return (appGallery || { default: {} }).default
       default:
         return APP_NOT_LOADED
     }
```

Comment:
It looks like something that should be done using a dictionary, but feel free to resolve this comment without doing anything since you only did some linting here.